Project 2/src/calculator/ConsoleCalculator.java

```diff
@@ -0,0 +1,33 @@
+package calculator;
+
+import java.util.Scanner;
+
+public class ConsoleCalculator {
+    public static int operation(int Number1, String operation, int Number2) {
```

Comment:
Number1 and Number2 names do not meet naming requirements described in Java code convention, please rename